src/konnektor/network_planners/generators/_abstract_network_generator.py

```diff
@@ -29,27 +29,28 @@ def __init__(
         progress: bool = False,
         _initial_edge_lister=None,
     ):
-        """This class is an implementation for the LigandNetworkPlanner interface.
-        It defines the std. class for a Konnektor LigandNetworkPlanner
+        """This class is an implementation for the NetworkGenerator interface.
+        It defines the std. class for a Konnektor NetworkGenerator.
 
         Parameters
         ----------
-        mapper : AtomMapper
+        mappers : AtomMapper
             the AtomMappers to use to propose mappings.  At least 1 required,
             but many can be given, in which case all will be tried to find the
             lowest score edges
         scorer : AtomMappingScorer
-            any callable which takes a AtomMapping and returns a float
+            Any callable which takes a AtomMapping and returns a float
+        network_generator:
```

Comment:
Do we need to add the explanation here? (I'm not yet sure what the AbstractNetworkAlgorithm is, will have to check).

Reply:
this naming convention is confusing, I think we should rename this as a follow-up